metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -312,13 +316,17 @@ def run(
         if pipeline_path is None:
             raise CommandException("Please specify --pipeline-path")
 
-        pipeline_path = flow.create_workflow_yaml_file(
+        if kind not in ["Workflow", "ConfigMap"]:
+            raise CommandException("Please specify --kind=Workflow or --kind=ConfigMap")
```

Comment:
Could you please explain when `--kind=ConfigMap` should be used by a user? I assume concurrency should be specified through `max_run_concurrency`

Reply:
Good question.
added a help description!